extension/src/fileSystem/index.ts

```diff
@@ -214,31 +214,27 @@ const loadYamlAsDoc = (
   }
 }
 
-const getPlotYamlObj = (cwd: string, plot: PlotConfigData) => {
+const getPlotYamlObj = (plot: PlotConfigData) => {
   const { x, y, template } = plot
   const plotName = `${template}_plot`
   return {
     [plotName]: {
       template,
-      x: x.file === y.file ? x.key : { [relative(cwd, x.file)]: x.key },
-      y: { [relative(cwd, y.file)]: y.key }
```

Comment:
Updated `file` to be the relative path instead of the absolute path since we need the relative path for error messages and for quick pick labels in `pickPlotConfiguration`.